bindings/rust/wasmedge-sys/src/async_env.rs

```diff
@@ -0,0 +1,138 @@
+use std::{
```

Comment:
Please add a summary for this new module. As for the format, please refer to the existed. Thanks!

Reply:
In addition, please also add doc comments for all `public` types and interfaces. Thanks!